rplugin/python3/nvim_ipy/__init__.py

```diff
@@ -317,7 +337,7 @@ def ipy_run(self, args):
                     self.km.start_kernel(**self.km._launch_args)
             return
 
-        reply = self.waitfor(self.kc.execute(code,silent=silent))
+        reply = self.waitfor(self.kc.execute(code + self.append,silent=silent))
```

Comment:
does other clients (`jypyter console` is my reference) always make sure to end code in a `\n`? then we could do this unconditionally, without needing an option. (though I would add this to the line and visual run code, not here)

Reply:
No, it is purely QtConsole thing.